storage/objects/main.go

```diff
@@ -289,6 +289,33 @@ func rotateEncryptionKey(client *storage.Client, bucket, object string, key, new
 	return nil
 }
 
+func downloadUsingRequesterPays(client *storage.Client, object, bucketName, localpath, billingProjectID string) error {
+	ctx := context.Background()
+	// [START storage_download_file_requester_pays]
+	bucket := client.Bucket(bucketName).UserProject(billingProjectID)
+	src := bucket.Object(object)
+
+	f, err := os.OpenFile(localpath, os.O_RDWR|os.O_CREATE, 0755)
+	if err != nil {
+		return err
+	}
+	rc, err := src.NewReader(ctx)
+	if err != nil {
+		return err
+	}
+	if _, err := io.Copy(f, rc); err != nil {
+		return err
+	}
+	if err := rc.Close(); err != nil {
+		return err
+	}
+	fmt.Printf("Downloaded file.txt using %v as billing project.\n", billingProjectID)
```

Comment:
Question: should this message be more generic? For example

```
fmt.Printf("Downloaded file using %v as billing project.\n", billingProjectID)
```

I'm not sure if the user has an option to select which file can be downloaded.

Last comment everything else LGTM.